### DISCUSSION TITLE
Oscam, ErsatzTV, MediathekView

### DISCUSSION THREAD

**Discussion by rj-d2** [selfhst/icons#154] [New Icon Request]
Oscam: [Github](https://github.com/linuxserver/docker-oscam) [Icon](https://dashboard.snapcraft.io/site_media/appmedia/2021/04/oscam.png)

ErsatzTV: [Github](https://github.com/ErsatzTV/ErsatzTV) [Icon](https://raw.githubusercontent.com/ErsatzTV/ErsatzTV/refs/heads/main/artwork/Ersatztv.png)

MediathekView: [Github](https://github.com/mediathekview/MediathekView) [Icon](https://raw.githubusercontent.com/mediathekview/MediathekView/refs/heads/master/src/main/resources/mediathek/res/MediathekView.png)

**Comment by rj-d2** [comment]
TheLounge: [Github](https://github.com/thelounge/thelounge)  [Icon](https://raw.githubusercontent.com/thelounge/thelounge/refs/heads/master/client/img/logo-grey-bg-512x512px.png)

**Comment by shollyethan** [comment]
These have all been added - thanks!
